build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "viash"
 
-version := "0.7.1"
+version := "0.7.3"
```

Comment:
version := "0.7.2"

Reply:
Derp, glad somebody's paying attention :p can you surround your suggestion with a suggestion code block?

Reply:
Aha, I missed how that worked and thought it picked it up by itself. Should be fixed now

Reply:
:+1: